llvm/lib/TableGen/Error.cpp

```diff
@@ -40,12 +40,22 @@ static void PrintMessage(ArrayRef<SMLoc> Loc, SourceMgr::DiagKind Kind,
                         "instantiated from multiclass");
 }
 
+[[noreturn]] inline static void exit() {
```

Comment:
Please could you pick a more descriptive name than `exit`, something that hints at what it does.

Reply:
Will do

Reply:
Done.